setup_builder.py

```diff
@@ -325,7 +333,12 @@ def _build_out_sections(
 
     # [options] -- override/augment specific options
     cfg["options"]["python_requires"] = bsec.python_requires()
-    cfg["options"]["packages"] = "find:"  # NOTE: this finds all packages & sub-packages
+    packages = bsec.packages()
+    if packages:
+        cfg["options"]["packages"] = list_to_dangling(packages)
+    # else: this uses setuptools autodetection
+    #       https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#automatic-discovery
```

Comment:
The docs say this is experimental and possibly temporary, so it may be best to leave `find:` as the default.

Reply:
I checked the changelog, and I guess that was just added in v61, released this month :(

I'll switch this out

Reply:
Pypa's tools change so much it seems. That's both good and bad :smiley:

Reply:
Better now?  I also told it to ignore some common directories that aren't packages.